onmt/opts.py

```diff
@@ -658,6 +658,12 @@ def _add_decoding_opts(parser):
                    "sampling restricted to the k most likely next tokens. "
```

Comment:
random_sampling_topk defaulting to 1 will make random_sampling_top_p useless.
I can:
- default it to -1 or 0
- make an assertion when parsing the config : Assert random_sampling_top_p == 0 or random_sampling_topk != 1